CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 4.0.0 (June 29, 2021)
```

Comment:
Since we haven't released v `3.0.0` we can add these changes to version `3.0.0`

Reply:
Done